homeassistant/components/tuya/cover.py

```diff
@@ -50,21 +51,24 @@ class TuyaCoverEntityDescription(CoverEntityDescription):
             key=DPCode.CONTROL,
             name="Curtain",
             current_state=DPCode.SITUATION_SET,
-            current_position=DPCode.PERCENT_STATE,
+            current_position=DPCode.PERCENT_CONTROL,
+            current_position_fallback=DPCode.PERCENT_STATE,
```

Comment:
Let make this accept tuples instead
```suggestion
            current_position=(DPCode.PERCENT_CONTROL, DPCode.PERCENT_STATE),
```